oqs_scripts/try_handshake.py

```diff
@@ -96,20 +95,19 @@
 def try_handshake(bssl):
     random_sig = random.choice(sigs)
     server = subprocess.Popen([bssl, 'server',
-                                     '-accept', '0',
+                                     '-accept', '31416',
```

Comment:
Isn't this risky? If run on a test machine that coincidentally has this port taken, CI will fail "surprisingly", no?

Reply:
I don't believe any known services use this port: https://www.speedguide.net/port.php?port=31416.

Additionally, `ubuntu-latest` image does not have any services using this port, so it's safe.

Reply:
Thanks for the background check. It still "feels" unclean (I hate hard-coding stuff as that always carries the potential to "bite" me in the future). But a) you're using a mathematician's holy number, then of course, this is "Good" and b) it'll bite _you_ :-)